wp-instagram-widget.php

```diff
@@ -68,9 +68,13 @@ function widget( $args, $instance ) {
 		$target = empty( $instance['target'] ) ? '_self' : $instance['target'];
 		$link = empty( $instance['link'] ) ? '' : $instance['link'];
 
-		echo $args['before_widget'];
+		$output = '';
 
-		if ( ! empty( $title ) ) { echo $args['before_title'] . wp_kses_post( $title ) . $args['after_title']; };
+		$output .= $args['before_widget'];
+
+		if ( ! empty( $title ) ) {
+			$output .= $args['before_title'] . wp_kses_post( $title ) . $args['after_title'];
+		};
 
 		do_action( 'wpiw_before_widget', $instance );
```

Comment:
These actions may need some tweaking now? - if they are being used for ouput that will now break